src/ui/zcl_abapgit_free_sel_dialog.clas.abap

```diff
@@ -17,9 +17,12 @@ CLASS zcl_abapgit_free_sel_dialog DEFINITION
         text             TYPE rsseltext,
       END OF ty_free_sel_field,
       ty_free_sel_field_tab TYPE STANDARD TABLE OF ty_free_sel_field WITH DEFAULT KEY.
+
+    TYPES: ty_syst_title TYPE sychar70.
```

Comment:
oh oh @larshp won't like this ;-)

`type c length 70`

Reply:
controversial. But ok, fixed

Reply:
`syst_title` did not exist in 731, `sychar70` might not exist in ABC, ie. best to avoid both

Reply:
or make it `TYPE sy-title`, if its semantically a sy title